runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -344,15 +337,14 @@ public UUID getStreamID() {
      */
     @Override
     public <R> R TXExecute(Supplier<R> txFunction) {
-        boolean isEnabled = MetricsUtils.isMetricsCollectionEnabled();
-        try (Timer.Context context = MetricsUtils.getConditionalContext(isEnabled, timerTxn)) {
-            return TXExecuteInner(txFunction, isEnabled);
+        try (Timer.Context context = timerTxn.getContext()) {
+            return TXExecuteInner(txFunction);
         }
     }
 
     @Deprecated // TODO: Add replacement method that conforms to style
     @SuppressWarnings({"checkstyle:membername", "checkstyle:abbreviation"}) // Due to deprecation
-    private <R> R TXExecuteInner(Supplier<R> txFunction, boolean isMetricsEnabled) {
+    private <R> R TXExecuteInner(Supplier<R> txFunction) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'TXExecuteInner' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651478)